test/e2e/suit_test.go

```diff
@@ -53,6 +56,12 @@ import (
 const (
 	ReleaseName      = "spark-operator"
 	ReleaseNamespace = "spark-operator"
+
+	MutatingWebhookName   = "spark-operator-webhook"
+	ValidatingWebhookName = "spark-operator-webhook"
```

Comment:
It looks like these constants aren't used where I imagine you intended to use them on lines 148 and 149.

Reply:
You are right, I have fixed it.